src/main.rs

```diff
@@ -54,6 +54,10 @@ fn main() {
                 version: sub_matches.value_of("version").map(|version| {
                     input_version::InputVersion::from_str(version).expect("invalid version")
                 }),
+                configure_opts: match sub_matches.values_of("configure_opts") {
+                    Some(opts) => opts.map(move |opt| opt.to_string()).collect(),
```

Comment:
I think you don't need `move`.

```suggestion
                    Some(opts) => opts.map(opt| opt.to_string()).collect(),
```